android/commons/src/main/java/com/alibaba/weex/commons/adapter/ImageAdapter.java

```diff
@@ -41,65 +35,66 @@ public ImageAdapter() {
   @Override
   public void setImage(final String url, final ImageView view,
                        WXImageQuality quality, final WXImageStrategy strategy) {
-    Runnable runnable = new Runnable() {
-
-      @Override
-      public void run() {
-        if(view==null||view.getLayoutParams()==null){
-          return;
-        }
-        if (TextUtils.isEmpty(url)) {
-          view.setImageBitmap(null);
-          return;
-        }
-        if (null != strategy){
-          recordImgLoadAction(strategy.instanceId);
-        }
-
-        String temp = url;
-        if (url.startsWith("//")) {
-          temp = "http:" + url;
-        }
-
-        if(!TextUtils.isEmpty(strategy.placeHolder)){
-          Picasso.Builder builder=new Picasso.Builder(WXEnvironment.getApplication());
-          Picasso picasso=builder.build();
-          picasso.load(Uri.parse(strategy.placeHolder)).into(view);
-
-          view.setTag(strategy.placeHolder.hashCode(),picasso);
-        }
-
-        Picasso.with(WXEnvironment.getApplication())
-                .load(temp)
-                .transform(new BlurTransformation(strategy.blurRadius))
-                .into(view, new Callback() {
-                  @Override
-                  public void onSuccess() {
-                    if(strategy.getImageListener()!=null){
-                      strategy.getImageListener().onImageFinish(url,view,true,null);
-                    }
-                    recordImgLoadResult(strategy.instanceId,true,null);
-
-                    if(!TextUtils.isEmpty(strategy.placeHolder)){
-                      ((Picasso) view.getTag(strategy.placeHolder.hashCode())).cancelRequest(view);
-                    }
-                  }
-
-                  @Override
-                  public void onError() {
-                    if(strategy.getImageListener()!=null){
-                      strategy.getImageListener().onImageFinish(url,view,false,null);
-                    }
-                    recordImgLoadResult(strategy.instanceId,false,null);
-                  }
-                });
-      }
-    };
-    if(Thread.currentThread() == Looper.getMainLooper().getThread()){
-      runnable.run();
-    }else {
-      WXSDKManager.getInstance().postOnUiThread(runnable, 0);
-    }
+      Picasso.with(view.getContext()).load(url).into(view);
+//    Runnable runnable = new Runnable() {
```

Comment:
Why is this outcommented code left here?

Reply:
I change this ImageAdapter because the origin has many conditions about how to display image. It's in demo project, so not the key core code